mautic/mautic
# Change version regexp

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

We need use version of Mautic in format X.X.X-xxx-yyy (4.2.0-beta-xxx). This PR allow it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Just unit tests and code review




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10919"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
